docs/docker-env-setup.md

```diff
@@ -12,24 +12,25 @@ sudo cp ./* /usr/bin/
 ```
 
 2. Download nydus-snapshotter release tarball from [the release page](https://github.com/containerd/nydus-snapshotter/releases). 
-```
+```shell
 # Get the latest version. If this version does not work for you, you can try v0.6.0
 TAG=`curl -s https://api.github.com/repos/containerd/nydus-snapshotter/releases/latest | grep tag_name | cut -f4 -d "\""`
 wget https://github.com/containerd/nydus-snapshotter/releases/download/"$TAG"/nydus-snapshotter-"$TAG"-linux-amd64.tar.gz
 tar -xzvf nydus-snapshotter-"$TAG"-linux-amd64.tar.gz
 sudo install -D -m 755 bin/containerd-nydus-grpc /usr/local/bin
 
-wget -O /etc/nydus/nydusd-config.fusedev.json https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/nydusd-config.fusedev.json
-wget -O /etc/nydus/config.toml https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/config.toml
+sudo wget -O /etc/nydus/nydusd-config.fusedev.json https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/nydusd-config.fusedev.json
+sudo wget -O /etc/nydus/config.toml https://raw.githubusercontent.com/containerd/nydus-snapshotter/"$TAG"/misc/snapshotter/config.toml
 ```
 
-3. Download nydus image service release tarball from [the release page](https://github.com/dragonflyoss/nydus/releases). 
-```
+3. Download nydus image service release tarball from [the release page](https://github.com/dragonflyoss/image-service/releases). 
```

Comment:
`image-service` -> `nydus` :)

Reply:
The PR submitted to the stable/v2.2 branch is also related to image-service, not nydus. When searching for https://github.com/dragonflyoss/image-service, it redirects to https://github.com/dragonflyoss/nydus. Should it be changed to nydus then? And what about the stable/v2.2 branch?